Update intake page - intake notification feature (#758)
SEKOIA-IO/documentation: docs/xdr/features/collect/intakes.md

When several conditions match the same event, the smart description with the most conditions is used as you can see in the test section.

## Notification when an intake is not receiving events

> You can set notification when the intake does not receive events:

Reviewer: ```suggestion
You can set a notification when an intake stops receiving events:
```
